core/network/src/sync.rs

```diff
@@ -452,6 +463,46 @@ impl<B: BlockT> ChainSync<B> {
 		}
 	}
 
+	fn handle_ancestor_search_state(
+		state: AncestorSearchState<B>,
+		n: NumberFor<B>,
+		block_hash_match: bool,
+	) -> Option<(AncestorSearchState<B>, NumberFor<B>)> {
+		println!("ancestor search: {:?} {:?} {:?}", n, block_hash_match, state);
```

Comment:
A bunch of others `println`s in the file, maybe change them to `trace!` ?

Reply:
Sorry, I forgot to add `in progress` label